src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.Dummy.cs

```diff
@@ -16,6 +16,8 @@ public partial class CSharpSyntaxTree
     {
         internal sealed class DummySyntaxTree : CSharpSyntaxTree
         {
+            private const SourceHashAlgorithm ChecksumAlgorithm = SourceHashAlgorithm.Sha1;
```

Comment:
Does this tree actually get used in emit somewhere? Why isn't this "None" and/or default?

Reply:
It doesn't get emitted but we assert in SourceText that the algorithm is valid, not `None`.

Reply:
Oh, that's bizarre that we assert that.